### ISSUE TITLE
Radio buttons don't announce their context label

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/hhmi#412]
> **Migrated from GitLab**
> **Original Author:** Grigor Malo (`@grgml`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/hhmi/hhmi/-/issues/64

Going through the user profile page with a screen reader, when you reach the radio buttons, e.g. the section "Are you interested in serving as a reviewer", the label is not announced, but only the radio options Yes and No.

## Expected behaviour

Screen reader should announce the common label of the radio buttons' group

## Current behaviour

Screen reader only announces the radio alternative, without giving context

## Steps to reproduce

[Provide a link to a live example or screenshots, and the steps to reproduce this bug]

1. Sign up and visit the profile page with a screen reader on
2. Tab through the fields until you reach "Are you interested in serving as a reviewer?" radio buttons
3. You only hear "Yes" and "No", but no context is given about what you are choosing

## Environment

Tested with Chrome Screen Reader in Google Chrome. 

## Possible solution

Group the radio buttons inside a radiogroup and give it a label

[Not required. Suggest a fix for the bug]
